client/blocks/image-editor/index.jsx

```diff
@@ -141,20 +141,46 @@ const ImageEditor = React.createClass( {
 		this.props.setImageEditorFileInfo( src, fileName, mimeType, title );
 	},
 
+	convertBlobToImage( blob ) {
+		const { onDone } = this.props;
+
+		// Create a new image from the canvas blob
+		const transientImage = document.createElement( 'img' ),
+			transientImageUrl = URL.createObjectURL( blob ),
+			imageProperties = this.getImageEditorProps();
```

Comment:
This is completely stylistic and optional, but we usually prefer one per line eg:
```jsx
const transientImage = document.createElement( 'img' );
const transientImageUrl = URL.createObjectURL( blob );
const imageProperties = this.getImageEditorProps();
```

These groupings don't happen very often anyway. With `let`/`const` having block scope we no longer need to worry about variable hoisting and defining all `var`s at the top of a function, so usually these definitions happen as closely as possible to where we end up using it.